resources/Songs_&_Hymns_Of_Life/metadata/434 - Just Taste And See That The Lord Is Good.json

```diff
@@ -1,45 +0,0 @@
-{
-  "title": "Just Taste And See That The Lord Is Good",
```

Comment:
assuming this was a just test file?

Reply:
Actually I think it was a duplicate, so I deleted.